Anytype/Sources/PresentationLayer/Modules/HomeWidgets/Container/Submodule/HomeUpdateView.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import SwiftUI
+
+struct HomeUpdateView: View, Animatable {
+    
+    @State private var locationState = CGFloat(0)
+    
+    var body: some View {
+        ZStack(alignment: .leading) {
+            Text(Loc.Widgets.appUpdate)
+                .anytypeStyle(.previewTitle2Medium)
+                .foregroundColor(.Text.primary)
+                .padding(.horizontal, 46)
+                .padding(.vertical, 10)
+                .frame(maxWidth: .infinity)
+            Image(asset: .X18.updateApp)
+                .foregroundColor(.Text.primary)
+                .padding(.horizontal, 16)
+        }
+        .onAppear {
+            withAnimation(.easeInOut(duration: 4).delay(0.5)) {
+                locationState = 6
+            }
+        }
```

Comment:
The rationale behind initializing the location state with the value of 6 is rather obscure. Might we consider utilizing boolean or enumerated values for the client-side API, while employing numerical representations internally?

You can find example in `struct ConfettiOverlay: View {`

Also I Suggest to store this animation `.easeInOut(duration: 4).delay(0.5)` in extensions for ease of further reuse.

Reply:
I can try to change 6 to 2 that will be means - two full cycle. 0...1 is one full cycle like percent range.
animation used only in this case. I don't sure it will be reuse anywhere